src/main/kotlin/no/nav/familie/ef/søknad/mapper/kontrakt/BarnepassMapper.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.familie.ef.søknad.mapper.kontrakt
+
+import no.nav.familie.ef.søknad.mapper.tilDesimaltall
+import no.nav.familie.ef.søknad.mapper.tilLocalDate
+import no.nav.familie.ef.søknad.mapper.tilSøknadsfelt
+import no.nav.familie.kontrakter.ef.søknad.Barnepass
+import no.nav.familie.kontrakter.ef.søknad.BarnepassOrdning
+import no.nav.familie.kontrakter.ef.søknad.Periode
+import no.nav.familie.kontrakter.ef.søknad.Søknadsfelt
+import no.nav.familie.ef.søknad.api.dto.søknadsdialog.Barnepass as BarnepassDto
+
+object BarnepassMapper {
+
+    fun tilBarnepass(barnepass: BarnepassDto): Barnepass {
+        return Barnepass(årsakBarnepass = barnepass.årsakBarnepass?.tilSøknadsfelt(),
+                         barnepassordninger = Søknadsfelt("", barnepass.barnepassordninger.map { // TODO
+                             BarnepassOrdning(hvaSlagsBarnepassOrdning = it.hvaSlagsBarnepassOrdning.tilSøknadsfelt(),
+                                              navn = it.navn.tilSøknadsfelt(),
+                                              periode = Søknadsfelt("", Periode(it.periode.fra.tilLocalDate().month, // TODO
```

Comment:
@charliemidtlyng @miichu - label her?

Reply:
Tror egentlig vi ikke vi har label på periode, men at tidligere har brukt `periodeFra` og `periodeTil` direkte på objektet. Gjøre samme her, kanskje`

Reply:
Vi har jo label på periode.fra / periode.til siden de begge består av IDatoFelt begge to? Eller har jeg misforstått noe her?

Reply:
Vi har ikke label på objektnivået til "Periode" eller lister/arrays, men på spesifikt feltnivå. Og ettersom vi i frontend har litt annen struktur enn kontraktene så blir det "mangelfullt" i de tilfellene man abstraherer verdiene ett nivå opp og slår sammen periode.fra og periode.til til et nytt objekt "Periode" - som her.